layers/+lang/alda/packages.el

```diff
@@ -0,0 +1,30 @@
+;;; packages.el --- Alda Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2017 Sylvain Benner & Contributors
```

Comment:
The year is 2018 now 😸 It's a common 'issue': https://github.com/syl20bnr/spacemacs/pull/11440